godotproject/Global/CountupTimer.gd

```diff
@@ -0,0 +1,12 @@
+extends Node
+
+var time : float = 0
+var max_time : float = 9 * 60 + 59 + (59/60)
+
+func _process(delta):
+	time += delta
+	# Cap timer at whatever the max_time is (9:59:59)
```

Comment:
Is this per level or for the whole game? Also If we're having a hard time limit for when the rocket takes off, maybe it should be a count down.

Reply:
This is mostly to prevent the on-screen timer from displaying a time over 10 minutes, since idk what would happen in that case, I only designed it to fit the 5 digits it has now